superset-frontend/src/explore/components/ExploreChartHeader.jsx

```diff
@@ -202,7 +202,10 @@ export class ExploreChartHeader extends React.PureComponent {
             status={CHART_STATUS_MAP[chartStatus]}
           />
           <ExploreActionButtons
-            actions={this.props.actions}
+            actions={{
+              ...this.props.actions,
+              openPropertiesModal: this.openProperiesModal,
```

Comment:
typo in `this.openProperiesModal` . Should be `this.openPropertiesModal`

Reply:
Yeah, that was there already, but let me fix it. I also hate typos :)